examples/libs/bitcoinjs/taproot/main.ts

```diff
@@ -0,0 +1,266 @@
+import {
+  initEccLib,
+  networks,
+  script,
+  payments,
+  crypto,
+  Psbt
+} from "bitcoinjs-lib";
+import { broadcast, waitUntilUTXO } from "./blockstream_utils";
+import { ECPairFactory, ECPairAPI, TinySecp256k1Interface } from 'ecpair';
+import { Taptree } from "bitcoinjs-lib/src/types";
+import { witnessStackToScriptWitness } from "./witness_stack_to_script_witness";
+
+import { LEAF_VERSION_TAPSCRIPT, } from 'bitcoinjs-lib/src/payments/bip341';
+
+import { DfnsWallet } from '@dfns/lib-bitcoinjs'
+import { DfnsApiClient } from '@dfns/sdk'
+import { AsymmetricKeySigner } from '@dfns/sdk-keysigner'
+import dotenv from 'dotenv'
+
+const tinysecp: TinySecp256k1Interface = require('tiny-secp256k1');
+initEccLib(tinysecp as any);
+const ECPair: ECPairAPI = ECPairFactory(tinysecp);
+const network = networks.testnet;
+
+dotenv.config()
+
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    credId: process.env.DFNS_CRED_ID!,
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  return DfnsWallet.init({ walletId, dfnsClient })
+}
+
+async function start() {
+  const wallet = await initDfnsWallet(process.env.BITCOIN_WALLET_ID!)
+
+  await start_p2pktr(wallet);
+  await start_taptree(wallet);
+}
+
+async function start_p2pktr(wallet: DfnsWallet) {
+  console.log(`Running "Pay to Pubkey with taproot example"`);
+  // Generate an address from the tweaked public key
+  const p2pktr = payments.p2tr({
+    pubkey: toXOnly(wallet.publicKey),
+    network
+  });
+  const p2pktr_addr = p2pktr.address ?? "";
+  console.log(`Waiting till UTXO is detected at this Address: ${p2pktr_addr}`);
+
+  const utxos = await waitUntilUTXO(p2pktr_addr)
+  console.log(`Using UTXO ${utxos[0].txid}:${utxos[0].vout}`);
+
+  const psbt = new Psbt({ network });
+  psbt.addInput({
+    hash: utxos[0].txid,
+    index: utxos[0].vout,
+    witnessUtxo: { value: utxos[0].value, script: p2pktr.output! },
+    tapInternalKey: toXOnly(wallet.rootPublicKey)
+  });
+
+  psbt.addOutput({
+    address: "tb1plaw5a9rpyezadkma95ghqmwjcmp9w0t8fyvgfx52m3hn2pqu0t8qwtzhpp", // faucet address
+    value: utxos[0].value - 200
+  });
+
+  await psbt.signInputAsync(0, wallet);
+  psbt.finalizeAllInputs();
+
+  const tx = psbt.extractTransaction();
+  console.log(`Broadcasting Transaction Hex: ${tx.toHex()}`);
+  const txid = await broadcast(tx.toHex());
+  console.log(`Success! Txid is ${txid}`);
+}
```

Comment:
I like to keep example focused. this first function is just a simple transfer with a taproot key. basically the same as the `basic-tx` example, not very interesting. we should just drop it.